Utils/Dataflow/019_esFormat/README

```diff
@@ -20,14 +23,16 @@ Required:
 
 Optional:
 * '_parent' (ES identifier for parent type record).
+* '_update' (corresponding ES record shouldbe "updated", not
+             "inserted"/"indexed")
```

Comment:
Missing "." at the end, to be in line with the previous element of the list.

Reply:
Fixed, thank you.